docs/software/index.md

```diff
@@ -13,10 +13,10 @@ The software has minimal branding, and the fonts and color scheme should work wi
 
 ## Free software
 
-VoteAmerica's software is free for anyone to use. You can find the code to embed it on your website on the [VoteAmerica website](https://www.voteamerica.com/free-software/).
+VoteAmerica's software is free for anyone to use. You can find the code to embed it on your website on the [VoteAmerica website](https://www.voteamerica.org/free-software/).
```

Comment:
Also this one - let's link to https://secure.voteamerica.com/free-software/

Reply:
Done, thank you!